proposals-CG-WG.md

```diff
@@ -29,70 +29,73 @@ The purpose of Stage 0 proposals is to allow anyone to raise and explore the Pro
-    * [ ] Working Group consensus to adopt the proposal as the basis for their work as a [Working Draft](https://www.w3.org/policies/process/#RecsWD).
-    * [ ] Working Group identification of the list of (seemingly resolvable) issues that have to be addressed before [Stage 3](#stage-3).
-    * [ ] An [Editor's Draft](https://www.w3.org/policies/process/#editors-draft) can be used to get PRs merged between [Working Draft](https://www.w3.org/policies/process/#RecsWD) revisions.
+    * [ ] Working Group consensus to adopt the proposal as the basis for their work.
+    * [ ] A clear list of blocking issues to be addressed before advancing to Stage 3.
+    * [ ] A completed Working Draft for further iteration.
+    * [ ] Approval to transition the draft into a formal as the basis for their work as a [Working Draft](https://www.w3.org/policies/process/#RecsWD).
+
   
 # Stage 3: Implementation
 
 > NOTE: We don't yet have enough experience as we get towards Stage 3 and 4, so they aren't fully baked yet. It is possible that these
-> will change a lot as we learn, including potentially entirely replacing them with references to the W3C Process.
+> will change a lot as we learn, including potentially entirely replacing them with references to the W3C Process. 
 
-The purpose of Stage 3 Proposals is to increase implementation and deployment confidence in order to produce a [Candidate Recommendation](https://www.w3.org/policies/process/#RecsCR).
+The purpose of Stage 3 Proposals is to increase implementation and deployment confidence in order to produce a [Candidate Recommendation](https://www.w3.org/policies/process/#RecsCR). The spec should be considered finished, pending editorial nit review. However, since multiple implementations are expected in this stage, it is possible there will be further changes to normative content.
  
   * What's needed from champions?
-    * [ ] Proposal fully merged into the [Working Draft](https://www.w3.org/policies/process/#RecsWD)
-    * [ ] Web Platform Tests
-    * [ ] Further implementation experience
+    * [ ] Proposal fully merged into the [Working Draft](https://www.w3.org/policies/process/#RecsWD).
+    * [ ] Web Platform Tests are available.
+    * [ ] At least two independent implementers and no unresolved objections (where resolution may include an agreement not to address the issue).
   * What's asked of the **Working Group**? 
-    * [ ] Working Group consensus that the [Working Draft](https://www.w3.org/policies/process/#RecsWD) sufficiently resolves all of the issues raised at [Stage 2](#stage-2)
-    * [ ] Working Group consensus to publish the [Working Draft](https://www.w3.org/policies/process/#RecsWD) as the Working Group's [Candidate Recommendation](https://www.w3.org/policies/process/#RecsCR)
+    * [ ] Working Group consensus that the [Working Draft](https://www.w3.org/policies/process/#RecsWD) sufficiently resolves all of the issues raised at [Stage 2](#stage-2).
```

Comment:
Since we are modifying this, maybe we should address the issue about not all features within a spec reaching Stage 3 at the same time. Perhaps the CR mentioned below can be a frozen branch from the WD, possibly with features in lesser stages removed?